zipline/gens/brokers/ib_broker.py

```diff
@@ -424,38 +425,46 @@ def portfolio(self):
         z_portfolio.cash = float(ib_account['TotalCashValue'])
         z_portfolio.start_date = None  # TODO(tibor)
         z_portfolio.positions = self.positions
-        z_portfolio.positions_value = None  # TODO(tibor)
-        z_portfolio.positions_exposure = None  # TODO(tibor)
+        z_portfolio.positions_value = float(ib_account['StockMarketValue'])
+        z_portfolio.positions_exposure = (z_portfolio.positions_value /
+                                          (z_portfolio.positions_value +
+                                           float(ib_account['TotalCashValue'])))
 
         return z_portfolio
 
     @property
     def account(self):
         ib_account = self._tws.accounts[self.account_id][self.currency]
+        portfolio = self.portfolio
```

Comment:
Could you please avoid this call?
Both used variables from portfolio exists in ib_account directly.